app/assets/javascripts/repository_navigation.js

```diff
@@ -29,12 +29,12 @@
 (function($) {
   $(function() {
     var revision = $('#revision-identifier-input'),
-      form = revision.closest('form'),
-      tag = $('#revision-tag-select'),
-      branch = $('#revision-branch-select'),
-      selects = tag.add(branch),
-      branch_selected = branch.length > 0 && revision.val() == branch.val(),
-      tag_selected = tag.length > 0 && revision.val() == tag.val();
+        form = revision.closest('form'),
+        tag = $('#revision-tag-select'),
+        branch = $('#revision-branch-select'),
+        selects = tag.add(branch),
+        branch_selected = branch.length > 0 && revision.val() == branch.val(),
+        tag_selected = tag.length > 0 && revision.val() == tag.val();
```

Comment:
Identifier 'tag_selected' is not in camel case.<br>Expected '===' and instead saw '=='.